requirements.txt

```diff
@@ -4,27 +4,18 @@
 #
 #    __main__.py compile -r -U
 #
-atomicwrites==1.3.0       # via pytest
-attrs==19.1.0             # via pytest
 beautifulsoup4==4.7.1
-coverage==4.5.3           # via pytest-cov
+coverage==7.4.1          # via pytest-cov
 docutils==0.14
-entrypoints==0.3          # via flake8
-flake8-commas==2.0.0
-flake8==3.7.7
-isort==4.3.18
-mccabe==0.6.1             # via flake8
-more-itertools==7.0.0     # via pytest
-multilint==2.4.0
-pluggy==0.11.0            # via pytest
-py==1.8.0                 # via pytest
-pycodestyle==2.5.0        # via flake8
-pyflakes==2.1.1           # via flake8
 pygments==2.4.0
-pytest-cov==2.7.1
-pytest-django==3.4.8
-pytest==4.4.1
+pytest-cov==4.1.0
+pytest-django==4.8.0
+pytest==8.0.0
```

Comment:
Was there a particular reason why pytest and coverage needed to be updated or is it just to get the latest versions?

Reply:
Yeah, I think it has to do with the fact that django moved some of its core to `async` functions, and the old pytest version wasn't ready to handle that.

I also had some issues in newer django versions where the coverage wasn't collected properly unless updating `pytest-cov`

Reply:
Makes sense! Thanks for the context